legal/prepareDocuments.js

```diff
@@ -0,0 +1,93 @@
+require("babel-register");
+const moment = require("moment");
+const fs = require("fs");
+const { join } = require("path");
+const replaceString = require("replace-string");
+const gitRev = require("git-rev-sync");
+const { formatMoney, formatDate, formatMomentDate } = require("./formatters");
+
+const DOCUMENT_CONSTANTS = {
+  repoUrl: "git@github.com:Neufund/ico-contracts.git",
+  website: "https://commit.neufund.org"
+};
+
+async function prepareNeumarkTokenHolderAgreement({
+  neumarkContract,
+  companyAddress
+}) {
+  const neuDecimals = (await neumarkContract.decimals()).toNumber();
+
+  const tags = {
+    "repo-url": DOCUMENT_CONSTANTS.repoUrl,
+    "commit-id": gitRev.long(),
+    "neumark-sc-address": neumarkContract.address,
+    "signed-by-company-date": formatMomentDate(moment.utc()),
```

Comment:
this should get this value from contract as per documentation:  `Neumark::currentAgreement[1]`